library/tasks/Picard.wdl

```diff
@@ -80,19 +87,22 @@ task SortBamAndIndex {
 }
 
 task CollectMultipleMetrics {
-  File aligned_bam
-  File genome_ref_fasta
-  String output_basename
+  input {
+    File aligned_bam
+    File genome_ref_fasta
+    String output_basename
 
-  # runtime values
-  String docker ="quay.io/humancellatlas/secondary-analysis-picard:v0.2.2-2.10.10"
-  Int machine_mem_mb = 8250
-  # give the command 1 GiB of overhead
-  Int command_mem_mb = machine_mem_mb - 1000
-  Int cpu = 1
-  # use provided disk number or dynamically size on our own, with 10GiB of additional disk
-  Int disk = ceil(size(aligned_bam, "GiB") + size(genome_ref_fasta, "GiB") + 10)
-  Int preemptible = 5
+    # runtime values
+    String docker ="quay.io/humancellatlas/secondary-analysis-picard:v0.2.2-2.10.10"
+    Int machine_mem_mb = 8250
+    # give the command 1 GiB of overhead
+    Int command_mem_mb = machine_mem_mb - 1000
+    Int cpu = 1
+    # use provided disk number or dynamically size on our own, with 10GiB of additional disk
+    Int disk = ceil(size(aligned_bam, "GiB") + size(genome_ref_fasta, "GiB") + 10)
```

Comment:
Should this be below the input block like the other ones that depend on inputs?

Reply:
I don't know, @pshapiro4broad ? that's something Bo's team did. I thihg everything should be under input, but I might be wrong. I'd like Phils input

Reply:
I think everything in the input block could be nice, that way you can easily override it from the inputs json if you need to change it